src/util/StdString.h

```diff
@@ -1,4 +1,6 @@
 #pragma once
+
+#if !defined(WINAPI_FAMILY_PARTITION) || WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
```

Comment:
IMO the include should be removed completely from the addon if it isn't needed, we shouldn't fix it here.